apps/dashboard/config/locales/en.yml

```diff
@@ -276,6 +276,11 @@ en:
       deleted_message: "Saved settings %{settings_name} deleted."
       settings_values_label: "Values"
       outdated_message: "%{app_title} parameters have changed since these settings were created."
+    
+    smart_attributes:
+      auto_log_location:
+        title: "Log Location"
+        description: "The location of the log file for this job."
```

Comment:
Does this need to be nested like this? Or can it just be `auto_log_location_title` or possibly just 1 nest at `auto_log_location.title? I think `smart_attributes` is a bit too low level to expose here in the locales.

Also description doesn't seem to be used.

Reply:
fixed.